### PR TITLE
Mapping cities covered by buses

### PR DESCRIPTION
**Pull request by inouire**

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -3885,7 +3885,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 5162;St-Denis-près-Martel;st-denis-pres-martel;8759457;87594572;44.9459316;1.6663378;;FR;Europe/Paris;f;f;f;t;f;f;FRSDM;SDM;t;;f;8700466;f;;f;;f;;f;;f;;f;;;;f;;f;8700247;f;;f;FRSDM;f;f;;;;;;;;;;;;;;;;;;;
 5163;Surdon;surdon;8744456;87444562;48.6647685;0.1344097;;FR;Europe/Paris;f;f;f;t;f;f;FRSDN;SDN;t;;f;8700229;f;;f;;f;;f;;f;;f;;;;f;;f;8700783;f;;f;FRSDN;f;f;;;;;;;;;;;;;;;;;;;
 5164;St-Dizier Hôpital;st-dizier-hopital;8710747;87107474;48.65657;4.96873;5167;FR;Europe/Paris;f;f;f;t;f;f;FRSDP;;t;;f;8705033;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;サン＝ディジエ;생디지에;;;Сен-Дизье;;;聖迪濟耶
-5165;Sedan;sedan;;;49.7000000000;4.9500000000;;FR;Europe/Paris;t;f;f;f;f;f;FRSDV;;t;;f;;f;;f;9558;t;;f;;f;;f;;;;f;;f;;f;;f;;f;t;;;;;;;;;;;;スダン;;;;Седан;;;
+5165;Sedan;sedan;;;49.7000000000;4.9500000000;;FR;Europe/Paris;t;f;f;f;f;f;FRSDV;;t;;f;;f;u0gd8n;t;9558;t;;f;;f;;f;;;;f;;f;;f;;f;;f;t;;;;;;;;;;;;スダン;;;;Седан;;;
```
Comment (avandecreme) [inline stations.csv]: `49.7000000000;4.9500000000` 👌

Comment (inouire) [reply stations.csv]: yep...

File: stations.csv [hunk 2]
```diff
@@ -5655,7 +5655,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 7264;Saulgrub;saulgrub;8020528;;47.666092;11.025395;;DE;Europe/Berlin;f;f;f;t;f;f;DEAWA;;f;;f;8005302;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;Заульгруб;;;绍尔格鲁布
 7265;St Michaelisdonn;st-michaelisdonn;8001505;;53.990075;9.117066;;DE;Europe/Berlin;f;f;f;t;f;f;DEAWB;;f;;f;8005655;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DEAWB;f;f;;;;;;;;;;;;;;;;;;;
 7266;St Wendel;st-wendel;8025296;;49.467036;7.165301;;DE;Europe/Berlin;f;f;f;t;f;f;DEAWC;;f;;f;8005659;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DESWL;f;f;;;;;;;;;;;;;;;;;;;
-7267;Schüttorf;schuttorf;8021157;;52.316841;7.220612;;DE;Europe/Berlin;f;f;f;t;f;f;DEAWD;;f;;f;8005436;t;;f;2788;t;;f;;f;;f;;;;f;;f;;f;;f;DEAWD;f;f;;;;;;;;;;;;;;;;Шютторф;;;许托尔夫
+7267;Schüttorf;schuttorf;8021157;;52.316841;7.220612;;DE;Europe/Berlin;f;f;f;t;f;f;DEAWD;;f;;f;8005436;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DEAWD;f;f;;;;;;;;;;;;;;;;Шютторф;;;许托尔夫
```
Comment (avandecreme) [inline stations.csv]: What happened to this one?

Comment (inouire) [reply stations.csv]: Not covered any more !

**Comment by avandecreme** [comment]
Could you remove the `Fix things` commit and make those fixes in the previous commits?

**Pull request merged**